packages/tools/devtools/devtools-browser-extension/package.json

```diff
@@ -152,8 +152,7 @@
 		"tasks": {
 			"compile": {
```

Comment:
I _think_ we can remove this entire `compile` block, since I believe this now matches the default configuration. Definitely verify that removing it doesn't change the build behavior, though. I could be wrong. @jason-ha maybe you can confirm.

Reply:
Yeah, the "..." bit basically means ["anything this task depends on by default in the global config"](https://github.com/microsoft/FluidFramework/blob/115c8f4e5c0b9dad79bd3932c417e865784affbc/build-tools/packages/build-tools/src/common/fluidTaskDefinitions.ts#L23) so the whole "compile" block is redundant now. I think it's correct to delete it, and with it the whole "fluidBuild" block can go since there would be no overrides to anything.

Reply:
Yep - remove